specs/README.md

```diff
@@ -0,0 +1,54 @@
+# Project Reunion Specs
+
+This directory contains archived and in-progress spec documents for APIs in
+Project Reunion.
+
+Documents in this directory are used as part of a feature and API reiview
+for features in development and are not intended to be final developer
+documentation.
+
+## Please DO:
+
+* Provide feedback by **commenting on active pull requests** tagged with
+    [the "ApiReview" label.](https://github.com/microsoft/ProjectReunion/pulls?q=is%3Apr+is%3Aopen+label%3AApiReview)
+* Provide feedback by **opening [issues](https://github.com/microsoft/ProjectReunion/issues/new/choose)
+   regarding spec documents in the [reviewed](https://github.com/microsoft/ProjectReunion/tree/master/specs/reviewed) folder**.
+   Specs in the `reviewed` folder have been previously reviewed for
+   inclusion in Project Reunion.
+
+## Please DO NOT:
+
+* Provide feedback on spec documents in the `archive` folder since those
+    features have been deprecated or altered substantially.
+* Create unsolicited PRs that add new specs. To propose a new feature,
+    please follow the Project Reunion contribution process [described here](https://github.com/Microsoft/ProjectReunion/docs/contributor-guide.md)
+    and start by [opening a new issue](https://github.com/microsoft/ProjectReunion/issues/new/choose).
+
+## Spec workflow
+
+For Project Reunion team members, the spec and API design workflow is:
+
+1. Ensure your proposal is approved and in plan.
+
+2. Create a new working branch ```user/<username>/<feature>``` or work in
+   your fork of the Project Reunion repo.
+
+3. Create a new folder for your spec under the ```specs``` folder: ```/specs/<feature>```
+
+4. Author your spec using the [spec template](spec_template.md).
+   Please use relative links for images or other assets in the folder.
+
+5. When your spec is ready for review:
+
+   * Open a new pull request to merge your spec to the ```master``` branch
+   * Reference the issue thread in the pull request to link them together
+
+6. Provide adequate time for the community and Project Reunion members to
+   review the API spec file.
+
+7. Review requested changes with Project Reunion members and the Windows
+   API Design representative responsible for the area.
+
+8. Complete the pull request
```

Comment:
When does the file get into the "reviewed" folder?

Reply:
I think that's up for debate.  Ideally, we have a "document per API family" that evolves over time. Adding new members to `Microsoft.Waffle.Pancake` means updating `specs/microsoft.waffle.md` and getting it reviewed & approved. WinUI splits that into "reviewed specs" and "implemented specs" so you can have "shovel ready" specs to implement that are reviewed but not in the product yet.

The Windows process of having a "new spec per API iteration" can lead to loss of institutional memory - "why did we do this that way" is buried in some long-ago spec discussion thread, rather than going "blame says this commit, which is from this PR, which has these comments."  Or maybe these aren't _API_ specs at all, but _feature_ specs that interweave API design...

Reply:
I was mostly commenting on the "please DO" section says to put things in reviewed folder but from the workflow i didn't understand how they ended up there. 

Also we ran into an issue where we'd want to do internal review with the team and *then* broader API review. Now maybe we shouldn't do that in general but if folks wanted to then I think we should recommend that they do it in stages where the first PR is from a topic branch into another non-main topic branch and then the API review PR is from topic branch to main/master.

Reply:
> put things in the reviewed folder

Oho, good point. I was intentionally _not_ following that part of the WinUI flow, but I left the "DO open issues" part behind. Will fix those sections, thanks!

On the internal front, I think we'll just need to have an internal fork of the repo for special cases.  Ideally we just do everything here, but I can certainly see (and already see) some debates where we should reach internal consensus before proposing something.

Reply:
Actually in our case it wasn't even that we wanted an internal phase an an external phase, we just wanted a team review phase (which included community feedback) and then an "ok we think this is good and we want API review" feedback phase. The second one is where we'd add you and Raymond, for example.